### ISSUE TITLE
Event created with google calendar was not received 

### ISSUE THREAD

**Issue by asmaNode** [linagora/esn-frontend-calendar#615]
Senario:
Create an event in google calendar and invite an openpaas user

**Comment by asmaNode** [comment]
Create an event with google calendar : (event not visible in openpaas calendar)

![eventfromgooglecalendar](https://user-images.githubusercontent.com/43316837/141947942-c535c39c-826c-46b4-9300-2f2e072ab8dd.gif)

accept an event: (event visible in openpaas calendar)
![eventfromcalendarAccepted](https://user-images.githubusercontent.com/43316837/141948318-2cf6919c-4379-452b-9d1a-9374b570632f.gif)



In saber there is no specific processing for the event created by an external user, the same event topic triggered (calendar:event:created) for internal or external user , so the problem normally : james does not send an event when creating

@BenoitTallandier WDYT?

**Comment by fabienmoyon** [comment]
It's not @BenoitTallandier, it's @chibenwa for James

**Comment by chibenwa** [comment]
Can we have the EML of such an event for unit testing?

**Comment by chibenwa** [comment]
Can we audit also the content of the queues of the RabbitMQ backing sabre?

This can easily be done through RabbitMQ management web API

**Comment by asmaNode** [comment]
yes that what we should do the queues of rabbit mq will explain if the event is received or not , i need to test this in production .
this bug does not persist for all users (only my account for the moment)
